README.md

```diff
@@ -67,6 +67,53 @@ targets:
 Note that configuration files can be stored in any place, even in a local folder. We recommend to
 use a VCS (like git) to store them; treat them as source code.
 
+### Using Docker to build and run Copybara
+
+You can build copybara using Docker like so
```

Comment:
Could you add a paragraph/warning saying that this is experimental?

Reply:
Addressed in #45